Dockerfile

```diff
@@ -1,14 +1,16 @@
 # This file is part of REANA.
-# Copyright (C) 2017, 2018 CERN.
+# Copyright (C) 2017, 2018, 2019, 2020 CERN.
```

Comment:
Minor: Actually this file was not touched in 2019:

```console
$ git log -- Dockerfile | grep ^Date
Date:   Sun Oct 7 17:52:30 2018 +0200
Date:   Mon May 29 14:00:52 2017 +0200
Date:   Thu Feb 16 11:17:18 2017 +0100
Date:   Fri Jan 27 10:19:21 2017 +0100
Date:   Thu Jan 19 09:41:23 2017 +0100
```

So you can add only 2020.